www-client/seamonkey/seamonkey-2.53.10.2.ebuild

```diff
@@ -0,0 +1,557 @@
+	# https://bugzilla.mozilla.org/show_bug.cgi?id=1623054
+	eapply "${FILESDIR}/${PN}-2.53.10.2-ownertab.patch"
+
+	# Shell scripts sometimes contain DOS line endings; bug 391889
+	grep -rlZ --include="*.sh" $'\r$' . |
+	while read -r -d $'\0' file ; do
+		einfo edos2unix "${file}"
+		edos2unix "${file}"
+	done
+
+	use system-libvpx \
+		&& eapply -p2 "${WORKDIR}/${PN}-2.53.3-system_libvpx-1.8.patch"
+
+	# Allow user to apply any additional patches without modifing ebuild
+	eapply_user
+
+	# Don't error for format with gcc-9
+	grep -rl -- '-Werror=format' | xargs sed -i 's/error=format/no-&/' || die
+
+	# Enable gnomebreakpad
+	if use debug ; then
+		sed -i -e "s:GNOME_DISABLE_CRASH_DIALOG=1:GNOME_DISABLE_CRASH_DIALOG=0:g" \
+			build/unix/run-mozilla.sh || die
+	fi
+
+	# Ensure that are plugins dir is enabled as default
+	sed -i -e "s:/usr/$(get_libdir)/mozilla/plugins:/usr/$(get_libdir)/${PN}/plugins:" \
+		xpcom/io/nsAppFileLocationProvider.cpp || die
+
+	# Don't exit with error when some libs are missing which we have in
+	# system.
+	sed '/^MOZ_PKG_FATAL_WARNINGS/s@= 1@= 0@' \
+		-i comm/suite/installer/Makefile.in || die
+	# Don't error out when there's no files to be removed:
+	sed 's@\(xargs rm\)$@\1 -f@' \
+		-i toolkit/mozapps/installer/packager.mk || die
+
+	# Don't build libs-% locale files for chatzilla if we are not building chatzilla
+	# (this is hard-coded in the build system at present rather than being based on configuration)
+	if ! use chatzilla ; then
+		sed '/extensions\/irc\/locales libs-/s@^@#@' \
+			-i comm/suite/locales/Makefile.in || die
+	fi
+
+	eautoreconf old-configure.in
+	cd js/src || die
+	eautoconf old-configure.in
+}
+
+src_configure() {
+	MEXTENSIONS="default"
+	# Google API keys (see http://www.chromium.org/developers/how-tos/api-keys)
+	# Note: These are for Gentoo Linux use ONLY. For your own distribution, please
+	# get your own set of keys.
+	_google_api_key=AIzaSyDEAOvatFo0eTgsV_ZlEzx0ObmepsMzfAc
+
+	######################################
+	#
+	# mozconfig, CFLAGS and CXXFLAGS setup
+	#
+	######################################
+
+	mozconfig_init
+
+	##################################
+	# Former mozconfig_config() part #
+	##################################
+
+	# Migrated from mozcoreconf-2
+	mozconfig_annotate 'system_libs' --with-system-bz2
+	mozconfig_annotate 'system_libs' --with-system-zlib
+
+	# Disable for testing purposes only
+	mozconfig_annotate 'Upstream bug 1341234' --disable-stylo
+
+	# Must pass release in order to properly select linker via gold useflag
+	mozconfig_annotate 'Enable by Gentoo' --enable-release
+
+	# Must pass --enable-gold if using ld.gold
+	if tc-ld-is-gold ; then
+		mozconfig_annotate 'tc-ld-is-gold=true' --enable-gold
+	else
+		mozconfig_annotate 'tc-ld-is-gold=false' --disable-gold
+	fi
+
+	# Enable position independent executables
+	mozconfig_annotate 'enabled by Gentoo' --enable-pie
+
+	mozconfig_use_enable debug
+	mozconfig_use_enable debug tests
+	if ! use debug ; then
```

Comment:
Can I suggest inverting this to put 'if use debug' first (and swap the order of the statements below)?